lib/libutils/isoc/fputs.c

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2020, Huawei Technologies Co., Ltd
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <trace.h>
+
+int fputs(const char *s, FILE *stream)
+{
+	if (stream != stdout && stream != stderr)
+		abort();
+
+	trace_ext_puts(s);
```

Comment:
`puts()` would be more accurate, `trace_ext_puts()` doesn't add a `\n` at the end.

Reply:
`fputs()` doesn't add a `\n` either...?

Reply:
Aha, you're correct, my mistake.